helm/hpcc/templates/dali.yaml

```diff
@@ -34,15 +34,20 @@ spec:
         volumeMounts:
 {{ include "hpcc.addConfigMapVolumeMount" $dali | indent 8 }}
 {{ include "hpcc.addDaliVolumeMount" $commonCtx | indent 8 }}
-{{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" (list "system" ) ) | indent 8 }}
+{{- $secretsCategories := list "system" }}
 {{- range $sashaName, $_sasha := $dali.services -}}
 {{- $sasha := ($_sasha | default dict) -}}
+{{- $secretsCategories2 := list "system" }}
 {{- if not $sasha.disabled -}}
 {{- $_ := set $sasha "name" $sashaName -}}
 {{- with ($sasha | merge (dict "access" (splitList " " (include "hpcc.getSashaServiceAccess" $sasha)))) }}
+{{- if or (has "data" .access) (has "dalidata" .access) -}}
+{{- $secretsCategories2 := append $secretsCategories2 "storage" -}}
+{{- end }}
+{{ include "hpcc.addSecretVolumeMounts" (dict "root" $ "categories" $secretsCategories2 ) | indent 8 }}
```

Comment:
this doesn't look right, it's now inside the range and will be repeated(duplicated) per sasha service.
(if you were to add 2 sasha services under dali.services, it would fail I think).

The line that was on line 37, was for the dali container volumeMounts (just "system").
The line that was on line 50, was for volumeMounts for each separate sasha service container.